src/coreclr/scripts/superpmi-setup.py

```diff
@@ -23,12 +23,12 @@
 # 4.  Lastly, it sets the pipeline variables.
 
 # Below are the helix queues it sets depending on the OS/architecture:
-# | Arch  | windows       | Linux                                                                                                                                |
-# |-------|------------------|--------------------------------------------------------------------------------------------------------------------------------------|
-# | x86   | Windows.10.Amd64 | -                                                                                                                                    |
-# | x64   | Windows.10.Amd64 | Ubuntu.1804.Amd64                                                                                                                    |
-# | arm   | -                | (Ubuntu.1804.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440 |
-# | arm64 | Windows.10.Arm64 | (Ubuntu.1804.Arm64)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-a45aeeb-20190620155855 |
+# | Arch  | windows                           | Linux                                                                                                                                |
+# |-------|-----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+# | x86   | Windows.10.Amd64.CoreAppCompat.Pr | -                                                                                                                                    |
```

Comment:
The name of the new queue to be used is Windows.10.Amd64.X86

Reply:
done.